src/status_im/navigation/events.cljs

```diff
@@ -119,6 +119,7 @@
   {:events [:show-bottom-sheet]}
   [{:keys [db]} content]
   (let [{:keys [sheets hide?]} (:bottom-sheet db)]
+    (rf/dispatch [:dismiss-keyboard])
     (rf/merge {:db (update-in db [:bottom-sheet :sheets] #(conj % content))}
```

Comment:
Please add `dispatch` according to [guideline](https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#registering-event-handlers)

Reply:
Ok